core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -2142,4 +2256,30 @@ public void setForceRetryFlag(boolean b) {
 	public void setNumberOfExceptionsCaughtWithoutMessageBeingReceivedThreshold(int number) {
 		this.numberOfExceptionsCaughtWithoutMessageBeingReceivedThreshold = number;
 	}
+
+	/**
+	 * After a message has an error in processing, there is a small delay before processing the next
+	 * message before processing the next message or retrying the failed message.
+	 * This is so that errors coming from external systems so not overload those external systems.
+	 * <p>
+	 *     The delay doubles after every failure, until the maximum set here is reached. If the transaction
+	 *     timeout can be determined, then the retry-delay is capped by half the transaction timeout to
+	 *     avoid messages automatically timing out.
+	 * </p>
+	 * <p>
+	 *     There is no delay after a message is successfully processed. After a message is successfully processed,
+	 *     the actual retry-delay is reset to 1 second.
+	 * </p>
+	 * <p>
+	 *     If set to 0, then there is no delay after messages that had an error.
+	 * </p>
+	 * <p>
+	 *     If this is not set on the receiver, then a default is taken from the configuration property {@code receiver.defaultMaxRetryDelay} which
+	 *     defaults to 100 seconds.
+	 * </p>
+	 * @param maxRetryDelay Maximum delay in seconds before retrying a message, after an error occurred during processing.
+	 */
+	public void setMaxRetryDelay(Integer maxRetryDelay) {
```

Comment:
```suggestion
	public void setMaxBackoffDelay(Integer maxRetryDelay) {
```
Misschien ook wel leuk te verwijzen naar https://en.wikipedia.org/wiki/Exponential_backoff ofzo?